hw3/03-react/game-of-thrones/src/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Route, Routes } from "react-router-dom";
+import "./App.css";
+import Home from "./views/Home";
+import Search from "./views/Search";
+import Houses from "./views/Houses";
+
+function Navbar() {
+  return (
+    <div>
+      <nav>
+        <ul className="Navbar">
+          <li>
+            <Link className="Navlink" to="/">
+              Home
+            </Link>
+          </li>
+          <li>
+            <Link className="Navlink" to="/search">
+              Search
+            </Link>
+          </li>
+          <li>
+            <Link className="Navlink" to="/houses">
+              Houses
+            </Link>
+          </li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<Home />} />
```

Comment:
Adding routes to separate file, would help if project scope is increased. Maybe App.js file can have all the routes.

Reply:
Thanks for the comments. I have addressed it.